auth/src/main/java/feast/auth/config/SecurityProperties.java

```diff
@@ -54,6 +54,6 @@
     private String provider;
 
     // K/V options to initialize the provider with
```

Comment:
this is no longer a k/V pair since we allow a list for roles.  may be worth a conversation